app/src/main/java/com/yoshitoke/weatheringwithyou/mvp/Contract.kt

```diff
@@ -0,0 +1,39 @@
+package com.yoshitoke.weatheringwithyou.mvp
+
+import com.yoshitoke.weatheringwithyou.mvp.model.DataClass.WeatherInfo
+
+interface Contract {
+    interface View {
+
+        fun showSpinnerList(data: List<String>)
+
+        fun showWeatherInfo(data: WeatherInfo)
+
+    }
+
+    interface Model {
+
+        interface OnWeatherFinishedListener {
+            fun onFinished(data: WeatherInfo?)
+        }
+
+        interface OnCityListFinishedListener {
+            fun onFinished(cityListName: List<String>)
+        }
+
+        interface OnFetchErrorListener {
+            fun onError(error: String?)
+        }
+
+    }
```

Comment:
remove all of this. return function into presenter to view. these interfaces are not necessary